src/server/scripts/Northrend/zone_grizzly_hills.cpp

```diff
@@ -569,6 +694,42 @@ class npc_wounded_skirmisher : public CreatureScript
     }
 };
 
+enum renewskirmisher
+{
+    NPC_WOUNDED_SKIRMISHER = 27463
+};
+
+class spell_renew_skirmisher : public SpellScriptLoader
+{
+public:
+    spell_renew_skirmisher() : SpellScriptLoader("spell_renew_skirmisher") { }
+
+    class spell_renew_skirmisher_SpellScript : public SpellScript
```

Comment:
use new script method please.

RegisterSpellScript

Reply:
https://www.azerothcore.org/wiki/core-scripts#stand-alone-spellscripts-and-aurascripts